test/fops/t.c

```diff
@@ -67,9 +70,11 @@ main(int argc, char *argv[])
 	};
 	u_int nthreads;
 	int ch, cnt, ret, runs;
-	char *config_open;
+	char *config_open, *working_dir;
```

Comment:
Can you change the name from ```working_dir``` to ```home```? We refer to it as the home directory in a bunch of places.